portfolio/src/main/webapp/script.js

```diff
@@ -22,6 +22,33 @@ function fetchData() {
   });
 }
 
+function fetchComments() {
+  fetch('/data').then(response => response.json()).then((commentData) => {
+    
+    //Test console log statements
+    console.log("Hi")
+    console.log(commentData);
+    
+    //Loops through elements of json object to get every comment
+    const comments = document.getElementById('comment-container');
+    comments.innerHTML = '';
+    for(i = 0; i < commentData.length; i++) {
```

Comment:
I think there's a shortcut for this in JavaScript like:

```
for (const comment in commentData) {
  comments.appendChild(...comment.author)); ... 
  ...
}
```

Or you can do it the functional way:

```
commentData.foreach(comment => {
  comments.appendChild(...comment.author)); ...
  ...
  }
);
```

The `...` above are just placeholders for your existing code because I'm too lazy to type it all again :)